include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3506,15 +3506,15 @@ struct OrtApi {
    * \param[in] options
    * \param[in] initializer_names Array of null terminated UTF-8 encoded strings of the initializers names.
    * \param[in] initializers Array of ::OrtValue type
-   * \param[in] initializers_num Number of elements in the initializer_names and initializers
```

Comment:
Can we add a comment, that this corresponds to the file buffers and names number and that we are feeding one initializer per file then?

Reply:
here's the for the existing API AddExternalInitializers. nothing to do with file buffers.
If you are talking about the new API, one file can contain multiple initializers.